diff --git a/node_modules/@urql/exchange-graphcache/CHANGELOG.md b/node_modules/@urql/exchange-graphcache/CHANGELOG.md
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/LICENSE b/node_modules/@urql/exchange-graphcache/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/README.md b/node_modules/@urql/exchange-graphcache/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/08e6d3e1.min.mjs b/node_modules/@urql/exchange-graphcache/dist/08e6d3e1.min.mjs
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/08e6d3e1.min.mjs.map b/node_modules/@urql/exchange-graphcache/dist/08e6d3e1.min.mjs.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/1cc90026.min.js b/node_modules/@urql/exchange-graphcache/dist/1cc90026.min.js
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/1cc90026.min.js.map b/node_modules/@urql/exchange-graphcache/dist/1cc90026.min.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/5301ccd2.mjs b/node_modules/@urql/exchange-graphcache/dist/5301ccd2.mjs
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/5301ccd2.mjs.map b/node_modules/@urql/exchange-graphcache/dist/5301ccd2.mjs.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/7832a774.js b/node_modules/@urql/exchange-graphcache/dist/7832a774.js
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/7832a774.js.map b/node_modules/@urql/exchange-graphcache/dist/7832a774.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/ast/index.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/ast/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/ast/node.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/ast/node.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/ast/schema.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/ast/schema.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/ast/schemaPredicates.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/ast/schemaPredicates.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/ast/traversal.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/ast/traversal.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/ast/variables.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/ast/variables.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/cacheExchange.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/cacheExchange.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/default-storage/index.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/default-storage/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/extras/index.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/extras/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/extras/relayPagination.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/extras/relayPagination.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/extras/simplePagination.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/extras/simplePagination.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/helpers/dict.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/helpers/dict.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/helpers/help.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/helpers/help.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/helpers/operation.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/helpers/operation.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/index.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/offlineExchange.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/offlineExchange.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/operations/index.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/operations/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/operations/invalidate.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/operations/invalidate.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/operations/query.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/operations/query.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/operations/shared.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/operations/shared.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/operations/write.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/operations/write.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/store/data.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/store/data.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/store/index.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/store/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/store/keys.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/store/keys.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/store/store.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/store/store.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/types/types.d.ts b/node_modules/@urql/exchange-graphcache/dist/types/types.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.js b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.js
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.js.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.min.js b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.min.js
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.min.js.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.min.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.min.mjs b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.min.mjs
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.min.mjs.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.min.mjs.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.mjs b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.mjs
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.mjs.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-default-storage.mjs.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.js b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.js
old mode 100644
new mode 100755
index f76f84e..810b159
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.js.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.min.js b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.min.js
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.min.js.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.min.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.min.mjs b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.min.mjs
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.min.mjs.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.min.mjs.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.mjs b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.mjs
old mode 100644
new mode 100755
index 659f2cd..2a0b44d
--- a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.mjs
+++ b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.mjs
@@ -172,7 +172,7 @@ function simplePagination(a) {
       for (q = new Set, n = [], k = null, x = 0; x < l; x++) {
         f = (h = m[x]).fieldKey;
         if (null !== (h = h.arguments) && e(c, h) && (f = p.resolveFieldByKey(a, f), h = h[g],
-        null !== f && 0 !== f.length && "number" == typeof h)) {
+        null !== f && 0 !== f.length && ("number" == typeof h || "string" == typeof h))) {
           for (y = [], z = 0; z < f.length; z++) {
             q.has(B = f[z]) || (y.push(B), q.add(B));
           }
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.mjs.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache-extras.mjs.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.js b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.js
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.js.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.min.js b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.min.js
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.min.js.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.min.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.min.mjs b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.min.mjs
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.min.mjs.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.min.mjs.map
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.mjs b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.mjs
old mode 100644
new mode 100755
diff --git a/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.mjs.map b/node_modules/@urql/exchange-graphcache/dist/urql-exchange-graphcache.mjs.map
old mode 100644
new mode 100755
