diff --git a/node_modules/react-native-sha256/android/build.gradle b/node_modules/react-native-sha256/android/build.gradle
index dae0077..7ccac38 100644
--- a/node_modules/react-native-sha256/android/build.gradle
+++ b/node_modules/react-native-sha256/android/build.gradle
@@ -1,12 +1,16 @@
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 28
-    buildToolsVersion "28.0.3"
+    compileSdkVersion  safeExtGet('compileSdkVersion', 28)
+    buildToolsVersion  safeExtGet('buildToolsVersion', "28.0.3")
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 28
+        minSdkVersion safeExtGet('minSdkVersion', 16)
+        targetSdkVersion safeExtGet('targetSdkVersion', 28)
         versionCode 1
         versionName "1.0"
         ndk {
@@ -21,4 +25,3 @@ android {
 dependencies {
     implementation 'com.facebook.react:react-native:+'
 }
-
